docs/the-top-10/missing_access_control.md

```diff
@@ -0,0 +1,49 @@
+# Missing Access Control
+
+Users have to fulfill different functions with devices. Not every user needs access to all functions. However, many (legacy) devices do not support authentication or authorization. As a result, unauthorized access is possible.
+
+## Description
+
+- Access control is needed on different levels.
+  - Access to the device/system in the plant from within the plant or company
+  - Access to the device/system in the plant from an external system -> remote access
+
+- old systems often have only limited authentication functions. they also offer no connection to a central identity management system. This makes it difficult to maintain and respond to changes in the organization's responsibilities on an ongoing basis
```

Comment:
could we give here a complete example for a legacy system with limited auth functions?